src/cockpit/389-console/src/lib/plugins/accountPolicy.jsx

```diff
@@ -490,7 +490,7 @@ class AccountPolicy extends React.Component {
                     console.info("accountPolicyOperation", "Result", content);
                     this.props.addNotification(
                         "success",
-                        cockpit.format(_("Config entry $0 was successfully $1ed"), configDN, action)
+                        cockpit.format(_("Config entry $0 was successfully $1"), configDN, action)
```

Comment:
The code seems generic 
  I suspect that action may be "add" and in this case it should display "added"
  IMHO you had better test explicitly for action == set

Reply:
FYI: I was thinking about something like:
 if (action === "set") {
      cockpit.format(_("Config entry $0 was successfully set"), configDN, action)
 } else {
      cockpit.format(_("Config entry $0 was successfully $1ed"), configDN, action)
   }
But if the function is called with "set" in all cases, then your code if fine !

Reply:
I think you can just do something like this:

```
cockpit.format(_("Config entry $0 was successfully $1"), configDN, action === "add" ? "added" : "set")
```

or assign it in to a const prior somewhere the line:

```
  const actionPastTense = action === "add" ? "added" : "set";
  this.props.addNotification(
                        "success",
                        cockpit.format(_("Config entry $0 was successfully $1"), configDN, actionPastTense)
                        );
 ```

Reply:
Thank you both.
I will apply these.